packages/compass-query-history/src/components/toolbar/toolbar.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback } from 'react';
+import { useId } from '@react-aria/utils';
```

Comment:
What is the benefit of using this library here rather than using some logic maintained by us? Looks like an extra dependency.

Reply:
We use react-aria for quite a few things already (for example to fix a11y issues of leafygreen tooltips) so it just made sense to use this dependency instead of maintaining more code on our own, this package would be a transitive dep anyway. What we probably should do is re-export it through compass-components so that we can make access to this useful helper easier without the need of having this package as a dependency everywhere

Reply:
Merging this pr then doing a follow up making those places that use `react-aria` use it from `compass-components` 👍